src/main/java/seedu/address/logic/commands/room/ListRoomCommand.java

```diff
@@ -0,0 +1,20 @@
+package seedu.address.logic.commands.room;
+
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+public class ListRoomCommand extends Command {
+    public static final String COMMAND_WORD = "listroom";
+    public static final String MESSAGE_SUCCESS = "Here are the list of rooms";
+    public static final String NUMBER_OF_ROOMS_UNDEFINED = "Please define number of rooms";
```

Comment:
maybe can change to: "there are no room in the app yet, please define no of rooms"

Reply:
Okay!

Reply:
To add on, can even add a "Please initialise the number of rooms using initRooms command".